### DISCUSSION TITLE
I have a question, how to text align center?

### DISCUSSION THREAD

**Discussion by hiepxanh** [Milkdown/milkdown#1303] [Q&A]
is there any possible to implement align center text? I already search and it seem not exist

**Comment by hiepxanh** [comment]
It's impossible with markdown since it's syntax does not support align :( sad

**Reply by quank123wip** [reply]
not quite right though, there are some plugins for remark which support these feature

**Comment by zgrybus** [comment]
@hiepxanh You may use css for it, I guess?
